src/main/webapp/mapPage.css

```diff
@@ -0,0 +1,9 @@
+#map {
+  height: 100%;
+}
+
+html, body {
+  height: 90%;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
I don't think I would recommend setting styles on the entire HTML document. Can you find a way to make this work without doing this? Also, what exactly is this doing?

Reply:
It was so that the map would be displayed, since I set the height of the map to 100% so it would take up the majority of the screen I needed to set the height of the whole page. I am planning on changing this and the CSS when I add the material.io formatting to the page with the sidebar in my next PR.

Reply:
ok. Can you add a TODO to clean thisup?